docs/source/en/main_classes/deepspeed.mdx

```diff
@@ -1853,52 +1853,47 @@ In this case you usually need to raise the value of `initial_scale_power`. Setti
 
 ## Non-Trainer Deepspeed Integration
 
-The [`~deepspeed.HfDeepSpeedConfig`] is used to integrate Deepspeed into the 🤗 Transformers core
+The `accelerate.state.AcceleratorState` is used to integrate Deepspeed into the 🤗 Transformers core
 functionality, when [`Trainer`] is not used. The only thing that it does is handling Deepspeed ZeRO-3 param gathering and automatically splitting the model onto multiple gpus during `from_pretrained` call. Everything else you have to do by yourself.
 
 When using [`Trainer`] everything is automatically taken care of.
 
-When not using [`Trainer`], to efficiently deploy DeepSpeed ZeRO-3, you must instantiate the
-[`~deepspeed.HfDeepSpeedConfig`] object before instantiating the model and keep that object alive.
+When not using [`Trainer`], to efficiently deploy DeepSpeed ZeRO-3, you must call static method
+`AcceleratorState.set_deepspeed_config` before instantiating the model.
 
-If you're using Deepspeed ZeRO-1 or ZeRO-2 you don't need to use `HfDeepSpeedConfig` at all.
+If you're using Deepspeed ZeRO-1 or ZeRO-2 you don't need to do anything.
 
 For example for a pretrained model:
 
 ```python
-from transformers.deepspeed import HfDeepSpeedConfig
+from accelerate.state import AcceleratorState
```

Comment:
Let's put `AcceleratorState` at the top level if we need to use it, so it's just
```suggestion
from accelerate import AcceleratorState
```